etc/profile-a-l/com.github.tchx84.Flatseal.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for flatseal
+# This file is overwritten after every install/update
+# Persistent local customizations
+include com.github.tchx84XXgg12uC.Flatseal.local
+# Persistent global definitions
+include globals.local
+
+# Required for GTK theem to work:
+read-only ${HOME}/.config/dconf/
+noblacklist ${HOME}/.config/dconf/
+whitelist ${HOME}/.config/dconf/
```

Comment:
```suggestion
```

Use `whitelist-commonc.inc`.

Reply:
Include it. Or you will not get an approving review from me. wc has been well tested across different distro/versions/configurations/… and non of these paths are really sensitive.

Reply:
Why grant access to read all these directories (and even write access to some of them) when it's not needed? Even if you haven't found any potential exploits, I really don't see a reason to give so much access.

I'd rather follow the principle of least privileged: to give access to strictly what's needed to do the job at hand, and nothing more.

Reply:
`whitelist-common.inc` seems to give read-write access to a bunch of locations, right. Shouldn't these be read-only? It seems that any application with those has permissions to mess with quite wide-scoped configurations files (e.g.: proxies used by KDE applications).